src/2_RTB/PianoDiProgetto/sections/ConsuntivoSprint/SecondoSprint.typ

```diff
@@ -0,0 +1,237 @@
+  [*Totale per ruolo*],[#ruoli_ore.at("Re")],[#ruoli_ore.at("Am")],[#ruoli_ore.at("An")],[#ruoli_ore.at("Pt")],[#ruoli_ore.at("Pr")],[#ruoli_ore.at("Ve")],[#ruoli_ore.values().sum()]
+)
+
+#figure(
+  image("../../assets/AreogrammiPartizioneOre/Effettivo/AreogrammaOreSecondoSprint.svg", width: 75%),
+  caption: [Areogramma della partizione delle ore per ruolo nel secondo sprint.],
+)
+
+#figure(
+  image("../../assets/IstogrammiOreMembro/Effettive/IstogrammaSecondoSprint.svg", width: 75%),
+  caption: [Istogramma delle ore svolte per persona nel secondo sprint.],
+)
+
+=== Rendicontazione costi a posteriori
+In questo sprint, il costo per ogni ruolo assunto viene riportato in tabella:
+#table(
+  columns: (120pt,60pt,100pt),
+  align: center,
+  [*Ruolo*],[*Ore*],[*Costo*],
+  [Responsabile],[#tab_costi.at("Re").at(0)],[#tab_costi.at("Re").at(1)],
+  [Amministratore],[#tab_costi.at("Am").at(0)],[#tab_costi.at("Am").at(1)],
+  [Analista],[#tab_costi.at("An").at(0)],[#tab_costi.at("An").at(1)],
+  [Progettista],[#tab_costi.at("Pt").at(0)],[#tab_costi.at("Pt").at(1)],
+  [Programmatore],[#tab_costi.at("Pr").at(0)],[#tab_costi.at("Pr").at(1)],
+  [Verificatore],[#tab_costi.at("Ve").at(0)],[#tab_costi.at("Ve").at(1)],
+  [*Totale*],[#ore_totali],[#costi_totali],
+  [*Rimanente*],[#ore_rimanenti],[#costi_rimanenti]
+)
+
+#figure(
+  image("../../assets/AreogrammiPartizioneCosti/Effettivo/AreogrammaCostiSecondoSprint.svg", width: 75%),
+  caption: [Areogramma del budget speso e rimanente preventivato per il secondo sprint.],
+)
+
+#figure(
+  image("../../assets/AreogrammiTotaliOre/Effettive/AreogrammaTotaleSecondoSprint.svg", width: 75%),
+  caption: [Areogramma delle ore svolte e rimanenti nel secondo sprint.],
+)
+
+=== Analisi a posteriori
+Il consuntivo, relativo allo sprint corrente, evidenzia significative disparità rispetto al preventivo, soprattutto in merito alla distribuzione dei ruoli. Tale deviazione è imputabile al cambiamento dei ruoli assegnati, avvenuto a metà sprint, circostanza non prevista durante la fase di pianificazione iniziale. Inoltre, va sottolineato che la conclusione delle attività pendenti dallo sprint precedente, ha determinato brevi periodi di tempo in cui i membri del team hanno mantenuto i ruoli assegnati nella fase precedente. Dato il considerevole aumento complessivo delle ore rispetto a quanto inizialmente previsto nel preventivo, diventa essenziale adottare un approccio più attento nella fase di pianificazione e apportare gli adeguamenti necessari ai futuri preventivi. In particolare, emerge una sottostima delle ore necessarie per il ruolo di Analist. Ciò è dovuto al fatto che, nel corso di questo sprint, il team si è focalizzato sulla stesura dell'_Analisi dei Requisiti_ e sulla sua ristrutturazione, intervenuta dopo il colloquio con il professor Cardin e la discussione con la Proponente. Queste considerazioni verranno utilizzate come base per ottimizzare la pianificazione degli sprint futuri.
```

Comment:
```suggestion
Il consuntivo, relativo allo sprint corrente, evidenzia significative disparità rispetto al preventivo, soprattutto in merito alla distribuzione dei ruoli. Tale deviazione è imputabile al cambiamento dei ruoli assegnati, avvenuto a metà sprint, circostanza non prevista durante la fase di pianificazione iniziale. Inoltre, va sottolineato che la conclusione delle attività pendenti dallo sprint precedente, ha determinato brevi periodi di tempo in cui i membri del team hanno mantenuto i ruoli assegnati nella fase precedente. Dato il considerevole aumento complessivo delle ore rispetto a quanto inizialmente previsto nel preventivo, diventa essenziale adottare un approccio più attento nella fase di pianificazione e apportare gli adeguamenti necessari ai futuri preventivi. In particolare, emerge una sottostima delle ore necessarie per il ruolo di Analista. Ciò è dovuto al fatto che, nel corso di questo sprint, il team si è focalizzato sulla stesura dell'_Analisi dei Requisiti_ e sulla sua ristrutturazione, intervenuta dopo il colloquio con il professor Cardin e la discussione con la Proponente. Queste considerazioni verranno utilizzate come base per ottimizzare la pianificazione degli sprint futuri.
```

invertirei questa frase:
" Ciò è dovuto al fatto che, nel corso di questo sprint, il team si è focalizzato sulla stesura dell'_Analisi dei Requisiti_ e sulla sua ristrutturazione, intervenuta dopo il colloquio con il professor Cardin e la discussione con la Proponente." 

in
"il team, dopo il colloquio con il prof cardin ........ Proponente, si è focalizzato sulla ...."